openeo/internal/jupyter.py

```diff
@@ -0,0 +1,61 @@
+from collections import Mapping
+import json
+
+# These classes are proxies to visualize openEO responses nicely in Jupyter
+# To show the actual list or dict in Jupyter, use repr() or print()
+
+SCRIPT_URL = 'https://cdn.jsdelivr.net/npm/@openeo/vue-components@2.0.0-rc.1/assets/openeo.js'
```

Comment:
maybe add a todo note here

     TODO: get this script url dynamically e.g. from a https://editor.openeo.org/ quert or from a local config file, with hardcoded fallback?

Reply:
I'm not sure. We could add it to a config file, but we can't and shouldn't get it through other means dynamically. The implementation is bound to the specific version and once we have a stable version 2.0.0 (in one or two weeks) it will be `SCRIPT_URL = 'https://cdn.jsdelivr.net/npm/@openeo/vue-components@2/assets/openeo.js'`, which will cover always the latest version of the 2.x branch and then we won't change it any time soon again as 3.x would be breaking anyway.

Reply:
We can keep this PR open until we have a 2.0.0 release.

Reply:
Or merge now? This will not be released immediately anyway, but it will make it available to other people testing on master?

Reply:
@jdries Sure, I'm fine with that. Just released another RC for vue-components so that all recent changes are in.

Reply:
> This will not be released immediately anyway

@jdries But five hours later 😂 👍